python_modules/libraries/dagster-k8s/dagster_k8s/pipes.py

```diff
@@ -485,6 +487,8 @@ def run(  # pyright: ignore[reportIncompatibleMethodOverride]
             ignore_containers (Optional[Set]): Ignore certain containers from waiting for termination. Defaults to
                 None.
             enable_multi_container_logs (bool): Whether or not to enable multi-container log consumption.
+            wait_pod_terminate_timeout (float): How long to wait for the pod to terminate before raising an exception.
+                Defaults to DEFAULT_WAIT_TIMEOUT. Set to 0 to disable.
```

Comment:
should say 24h / 1d instead of just referring to the const name here. Theres risk of dessync but the const name is just confusing if you are looking at the api docs on the website or IDE hover card

Reply:
Lets just keep the existing behavior in this PR to ensure it lands swiftly. You can open another for changing the default to 0 if you want. 

Anecdotally I have dealt with more user frustration from computations without timeouts getting stuck and overlooked and consuming resources for weeks/months than users hitting unexpected timeouts. All that to say I see the arguments both ways so its a harder call on changing the default.

Reply:
Ok, I'll put that in a separate PR. I feel very strongly about switching it because for one of our batch jobs it led to 100's of failure all over 24hrs. Each run for us is expensive, so we lost 24*100s of compute hrs on a run that failed due to a framework quirk costing us $1,000s of dollars as our application aren't journaled and we lost the compute with no good data to show for it.